modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view.jsp

```diff
@@ -326,7 +326,7 @@ KBArticleURLHelper kbArticleURLHelper = new KBArticleURLHelper(renderRequest, re
 						actionDropdownItems="<%= kbAdminManagementToolbarDisplayContext.getEmptyStateActionDropdownItems() %>"
 						animationType="<%= EmptyResultMessageKeys.AnimationType.EMPTY %>"
 						buttonCssClass="secondary"
-						title='<%= LanguageUtil.get(request, "knowledge-base-is-empty") %>'
+						title='<%= (parentResourcePrimKey == 0) ? LanguageUtil.get(request, "knowledge-base-is-empty") : LanguageUtil.get(request, "this-folder-is-empty") %>'
```

Comment:
it's better to use `KBFolderConstants.DEFAULT_PARENT_FOLDER_ID` than `0`

Reply:
That's true!